charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -22,7 +22,7 @@ sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
 version: 45.25.0
```

Comment:
Thanks for the PR! Please bump the chart version:
```suggestion
version: 45.26.0
```

Reply:
Thanks for review! Done. update chart version to `45.26.0`